Packs/HelloWorld/Playbooks/playbook-HelloWorld_Scan_README.md

```diff
@@ -0,0 +1,49 @@
+This Playbook simulates a vulerability scan using the "HelloWorld" sample integration. It's used to demonstrate how to use the GenericPolling mechanism to run jobs that take several seconds or minutes to complete. It is designed to be used as a subplaybook, but you can also use it as a standalone playbook, by providing the ${Endpoint.Hostname} input in the Context.
+
+Other inputs include the report output format (JSON context or File attached), and the Interval/Timeouts to use for polling the scan status until it's complete.
+
+## Dependencies
+This playbook uses the following sub-playbooks, integrations, and scripts.
+
+### Sub-playbooks
+* GenericPolling
+
+### Integrations
+This playbook does not use any integrations.
```

Comment:
this playbook actually uses integration HelloWorld.
And uses 3 commands

Reply:
We may have a bug here in generate-docs command

Reply:
fixed, will open a doc bug